internal/graphapi/loadbalancer.resolvers.go

```diff
@@ -6,8 +6,13 @@ package graphapi
 
 import (
 	"context"
+	"database/sql"
+	"fmt"
 
+	"github.com/labstack/gommon/log"
```

Comment:
This is pulling in an additional logging package

Reply:
✅